public/static/app/common/AllowlistDataService.test.js

```diff
@@ -1,12 +1,12 @@
-describe('WhitelistDataServiceSpec', function () {
+describe('AllowlistDataServiceSpec', function () {
   beforeEach(module('MCI'));
 
   const project = 'sys-perf';
   const variant = 'linux-standalone';
   const task = 'bestbuy_agg';
   const test_name = ' distinct_types_no_predicate-useAgg';
   const db_perf = 'perf';
-  const coll_whitelist = 'whitelisted_outlier_tasks';
+  const coll_allowlist = 'whitelisted_outlier_tasks';
```

Comment:
Note: The collection name here hasn't been updated. Other projects are using this collection so updating it will take some coordination. But I actually don't think this UI is even used any more, so I've opened EVG-14256 to confirm we are no longer using it and then we can come back and just delete all this code.